Can't download subtitle due to SSL error

- [X] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [X] I checked if the issue/feature exists in the latest version.
- [N/A] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

An error caused downloader to gets stuck then reporting error. Happened on 0.18.0 and 0.18.2
Can be tested with 8 bit guy or LGR videos that have subs 
https://www.youtube.com/user/adric22
 
My tablet is denver running 4.4.4

![Screenshot_2020-01-27-21-08-32](https://user-images.githubusercontent.com/24647715/73210429-9156ed80-414a-11ea-8eac-9a4c56f35cb2.png)

I have captured the logcat

```
01-27 21:08:25.570 E/DownloadInitializer(5782): initializer failed, retrying
01-27 21:08:25.570 E/DownloadInitializer(5782): javax.net.ssl.SSLHandshakeException: javax.net.ssl.SSLProtocolException: SSL handshake aborted: ssl=0x5bfdf748: Failure in SSL library, usually a protocol error
01-27 21:08:25.570 E/DownloadInitializer(5782): error:14077102:SSL routines:SSL23_GET_SERVER_HELLO:unsupported protocol (external/openssl/ssl/s23_clnt.c:714 0x57098ce4:0x00000000)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:448)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.Connection.upgradeToTls(Connection.java:146)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.Connection.connect(Connection.java:107)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:294)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpEngine.sendSocketRequest(HttpEngine.java:255)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:206)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:345)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:296)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpURLConnectionImpl.getResponseCode(HttpURLConnectionImpl.java:503)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.okhttp.internal.http.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:136)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at us.shandian.giga.get.DownloadMission.establishConnection(DownloadMission.java:249)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at us.shandian.giga.get.DownloadInitializer.run(DownloadInitializer.java:58)
01-27 21:08:25.570 E/DownloadInitializer(5782): Caused by: javax.net.ssl.SSLProtocolException: SSL handshake aborted: ssl=0x5bfdf748: Failure in SSL library, usually a protocol error
01-27 21:08:25.570 E/DownloadInitializer(5782): error:14077102:SSL routines:SSL23_GET_SERVER_HELLO:unsupported protocol (external/openssl/ssl/s23_clnt.c:714 0x57098ce4:0x00000000)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.org.conscrypt.NativeCrypto.SSL_do_handshake(Native Method)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	at com.android.org.conscrypt.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:405)
01-27 21:08:25.570 E/DownloadInitializer(5782): 	... 11 more

```

--- a/app/src/main/java/us/shandian/giga/get/DownloadMission.java
+++ b/app/src/main/java/us/shandian/giga/get/DownloadMission.java
@@ -223,6 +223,7 @@ public class DownloadMission extends Mission {
         conn.setInstanceFollowRedirects(true);
         conn.setRequestProperty("User-Agent", DownloaderImpl.USER_AGENT);
         conn.setRequestProperty("Accept", "*/*");
+        conn.setRequestProperty("Accept-Encoding", "*");
 
         if (headRequest) conn.setRequestMethod("HEAD");
 

